manager/apis/v2/types/schema.go

```diff
@@ -0,0 +1,123 @@
+package types
+
+type SchedulerCluster struct {
+	ClusterId       string `json:"cluster_id,size:63" binding:"omitempty"`
```

Comment:
Why use size 63?

Varchar  is 255 is useful value.

https://stackoverflow.com/questions/2340639/why-historically-do-people-use-255-not-256-for-database-field-magnitudes